.circleci/config.yml

```diff
@@ -0,0 +1,130 @@
+version: 2.1
+orbs:
+  ci: bigcommerce/internal@volatile
+  node: bigcommerce/internal-node@volatile
+
+#####################################################
+# Aliases
+#####################################################
+aliases:
+  - &node_executor
+      executor:
+        name: node/node
+        node-version: "10"
+
+  - &release_filter
+      branches:
+        only:
+          - master
+
+  - &pull_request_filter
+      branches:
+        ignore:
+          - master
+
+#####################################################
+# Commands
+#####################################################
+commands:
+  install:
+    steps:
+      - ci/pre-setup
+      - restore_cache:
+          keys:
+            - bigpay-client-js
+      - run: npm ci
+      - save_cache:
+          key: bigpay-client-js
+          paths:
+            - ~/.npm
+  test:
+    steps:
+      - run: npm run prerelease
+  build:
+    steps:
+      - run: npm run prebuild
+      - run: npm run build
+
+#####################################################
+# Jobs
+#####################################################
+jobs:
+  build-and-test:
+    <<: *node_executor
+    steps:
+      - install
+      - build
+      - test
+      - persist_to_workspace:
+          root: .
+          paths:
+            - lib
+      - store_artifacts:
+          path: coverage
+          destination: coverage
+  validate-commits:
+    <<: *node_executor
+    steps:
+      - ci/pre-setup
+      - ci/merge-base
+      - run:
+          name: 'Validate Commits'
+          command: '[ -z "$MERGE_BASE_SHA1" ] || npx @bigcommerce/validate-commits ${MERGE_BASE_SHA1}..${CIRCLE_SHA1}'
```

Comment:
It'd be nice to update our `internal-node` orb so it can reused for all projects. It has a `npm-validate-commits` job but I don't think it's working properly. 🍹